assets/css/abstracts/_colors.scss

```diff
@@ -20,3 +20,4 @@ $image-placeholder-border-color: #f2f2f2;
 $universal-border: rgba(17, 17, 17, 0.3); // Used for form step borders.
 $universal-border-light: rgba(17, 17, 17, 0.115); // e7e7e7 on white.
 $universal-body-low-emphasis: rgba(17, 17, 17, 0.5); // Used for low emphasis text such as input labels.
+$universal-border-dark: rgba(17, 17, 17, 0.8); // #1e1e1e on white. Used for input borders.
```

Comment:
This is fine for now but we may need to revisit naming depending on if it makes sense to target elements (-input-border) vs shades (-border-dark).